data.sql

```diff
@@ -138,3 +138,63 @@ BEGIN;
 UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
```

Comment:
- [ ] It would be better to implement some queries in our queries.sql file. For example; implement the following queries:
  - [ ] Who was the last animal seen by William Tatcher?
  - [ ] How many different animals did Stephanie Mendez see?
  - [ ] List all vets and their specialties, including vets with no specialties.
  - [ ] List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020
  - [ ] What animal has the most visits to vets?
  - [ ] Who was Maisy Smith's first visit?
  - [ ] Details for most recent visit: animal information, vet information, and date of visit.
  - [ ] How many times visits where with a vet that did not specialize in that animal's species?
  - [ ] What specialty should Maisy Smith consider getting? Look for the species she gets the most.
Once these are done. Kindly submit another code review request

Reply:
They were already in the queries file, however, I've gone ahead and added the descriptions to avoid any confusion.